odoo/addons/base/views/ir_ui_view_views.xml

```diff
@@ -106,31 +106,36 @@
             <field name="arch" type="xml">
                 <form string="Reset View Architecture">
                     <group>
-                        <field name="view_id" invisible="1"/>
-                        <field name="view_name"/>
-                        <field name="reset_mode" widget="radio"/>
+                        <group>
+                            <field name="view_id" invisible="1"/>
+                            <field name="view_name"/>
+                            <field name="view_to_id" attrs="{'invisible': [('reset_mode', '!=', 'other_view')]}" options="{'no_create': True, 'no_open': True}"/>
+                        </group>
+                        <group>
+                            <field name="reset_mode" widget="radio"/>
+                        </group>
                     </group>
                     <field name="arch_diff" attrs="{'invisible': [('arch_diff', '=', False)]}"/>
                     <div class="text-warning mb4 mt16" attrs="{'invisible': [('arch_diff', '!=', False)]}">
-                        <span attrs="{'invisible': [('reset_mode', '=', 'hard')]}">This view has no previous version.</span>
-                        <span attrs="{'invisible': [('reset_mode', '=', 'soft')]}">This view is not coming from a file.</span>
+                        <span attrs="{'invisible': [('reset_mode', '!=', 'soft')]}">This view has no previous version.</span>
+                        <span attrs="{'invisible': [('reset_mode', '!=', 'hard')]}">This view is not coming from a file.</span>
+                        <span attrs="{'invisible': [('reset_mode', '!=', 'other_view')]}">You need two views to compare.</span>
                     </div>
                     <footer>
-                        <button string="Reset View" name="reset_view_button" type="object" class="btn-primary" attrs="{'invisible': [('arch_diff', '=', False)]}"/>
-                        <button string="Reset View" name="reset_view_button" type="object" class="btn-primary" disabled="disabled" attrs="{'invisible': [('arch_diff', '!=', False)]}"/>
+                        <button string="Reset View" name="reset_view_button" type="object" class="btn-primary" attrs="{'invisible': ['|', ('arch_diff', '=', False), ('arch_diff', 'like', 'No Differences Found')]}"/>
```

Comment:
louch domain "No Differences Found"

compute methode never return it, and quid about translation ?